src/tests/helpers/mappingAdapter.ts

```diff
@@ -0,0 +1,23 @@
+const TEST_PORT = process.env.PORT || 50604
```

Comment:
I wasn't sure if these should be passed into the function or stated here. Passing them into the function would be sort of painful for the developer.

Reply:
this is okay. the user simply just do ```PORT=3095 npm run test``` to run the test if they want to run on different port.